nss-day-cohort-43/nuttyshell-the-real-housewives-of-transylvania
# News articles section

# Description
"Add a new article" modal is rendering properly. Still need to add additional functionality to it. 
Fixes # (issue)
## Type of change



# Testing Instructions
Run your json server. Log in or register. Refresh page. Click on Create New Article. The modal should open. You cannot save an entry yet. Click the X. The modal should close. 
# Checklist:
- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
